### PR TITLE
Update `ansible-lint` Configuration

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR updates the `ansible-lint` configuration to remove some `exclude_paths` items.

## 💭 Motivation and context ##

With the merge of #88 we no longer need to exclude `converge.yml` files (at this time).

Previously we were excluding the `molecule-no-systemd.yml` and `molecule-with-systemd.yml` files in the `molecule/default/` directory for our [ansible-lint] configuration. This worked well enough as long as those files only existed in the `default` scenario for [molecule] and were symlinked for other scenarios. However, [cisagov/ansible-role-samba] does not have a `default` scenario, and so those files existing in a different scenario caused it to fail the `ansible-lint` hook for [pre-commit]. The change in the configuration for this type of file now brings it in line with how [ansible-lint] handles [`molecule.yml` files by default](https://github.com/ansible-community/ansible-lint/blob/5a2923d77a3c84c8e09f38a2933d07502f5f9da0/src/ansiblelint/config.py#L34). This will ensure that no matter the scenario configuration for an Ansible role project, our `systemd` configurations for [molecule] should be safely checked.

## 🧪 Testing ##

Automated tests pass. I also made these changes in https://github.com/cisagov/ansible-role-samba/pull/3 so that it would pass tests.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

[ansible-lint]: https://github.com/ansible-community/ansible-lint
[molecule]: https://github.com/ansible-community/molecule
[pre-commit]: https://pre-commit.com
[cisagov/ansible-role-samba]: https://github.com/cisagov/ansible-role-samba

### CODE REVIEW

File: .ansible-lint [hunk 1]
```diff
@@ -13,10 +13,10 @@ exclude_paths:
...
-  # what molecule requires
-  - molecule/default/converge.yml
-  # These two are Molecule configuration files, not Ansible playbooks
-  - molecule/default/molecule-no-systemd.yml
-  - molecule/default/molecule-with-systemd.yml
+kinds:
+  # This will force our systemd specific molecule configurations to be treated
+  # as plain yaml files by ansible-lint. This mirrors the default kind
+  # configuration in ansible-lint for molecule configurations:
+  # yaml: "**/molecule/*/{base,molecule}.{yaml,yml}"
```
Comment (jsf9k) [inline .ansible-lint:20]: Why is this commented out line here?

**Review by jsf9k — COMMENTED** [review]
I have one question.  Also, should this PR have the `blocked` label?

File: .ansible-lint (hunk 1 above)
Comment (mcdonnnj) [reply .ansible-lint:20]: Sorry, I guess that commenting is unclear. That is showing how `ansible-lint` handles `molecule.yml` files internally. The description has [this link](https://github.com/ansible-community/ansible-lint/blob/5a2923d77a3c84c8e09f38a2933d07502f5f9da0/src/ansiblelint/config.py#L34) to where it's defined in the `ansible-lint` source.

**Comment by mcdonnnj** [comment]
Ugh, I just realized this should be done in [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic). @jsf9k Please see https://github.com/cisagov/skeleton-generic/pull/95 now.

**Pull request closed without merging**